README.md

```diff
@@ -4,27 +4,67 @@ Chemical Dataset Exploration Tool Built on Plotly Dash
 
 The chemdash app is a simple analysis application built upon the plotly dash framework.  As input the application takes a _csv_ file requiring both **smiles** and **Compound_id** fields, other fields will be retained and displayed in the data table portion of the application.  A UMAP (similar to t-SNE/PCA) plot will be generated as well as a Molecular Grid Map (MGM).   Depending on the column selection in the data table, a distribution plot for the given column will also be displayed.  Selection of data points in either the UMAP or MGM plot will result in filtering of the data table and highlighting within the distribution plot.  For more information on this demo app please check out [my blog post](https://cognitivedataworks.com/2020/02/building-a-gan-compound-explorer-in-plotly-dash).
 
-## Installation
+## Installation & Execution
 
-The included yml file should allow you to get started quite easily if you have anaconda installed.  
+Download the repository locally, then install the dependencies from the top of the git repo:
 
+### Docker
+
+This works on most platforms, including Mac on Apple processors:
+```
+docker buildx build --platform linux/amd64 -t chemdash:latest .
+```
+
+To start chemdash:
 ```
-conda env create -f environment.yml
+docker run --rm --name chemdash -p 8000:8000 -v /tmp:/tmp -v /var/tmp:/var/tmp chemdash:latest
 ```
 
+To stop it:
+```
+docker kill chemdash
+```
 
-## Execution
+### Non-docker install and run:
 
-Download or clone the repository locally, make sure your new environment is active and navigate to the directory containing _chemdash.py_ and run:
+A non-docker install can be useful for debugging.
 
+Install non-python dependencies on ubuntu (note 22.04 is required, since some modules in 24.04, with python 3.12, have problems.)
 ```
+./install-deps-ubuntu-22.04.sh
+```
```

Comment:
Install script for Linux.  This could be adapted to Ubuntu 24.04 but it will require an older python (3.10 instead of 3.12) until some packages are revised.